src/main/java/edu/tamu/iiif/exception/UnknownActionException.java

```diff
@@ -0,0 +1,12 @@
+package edu.tamu.iiif.exception;
```

Comment:
Don't need this exception.

Reply:
https://github.com/TAMULib/IRIIIFService/commit/b48dea90a4271025c9772504f717734f25893795